website/source/docs/commands/operator/rekey.html.md

```diff
@@ -47,6 +47,17 @@ $ vault operator rekey \
     -pgp-keys="keybase:hashicorp,keybase:jefferai,keybase:sethvargo"
 ```
 
+Rekey KMS recovery keys and encrypt the resulting keys with PGP:
```

Comment:
This is really every auto seal, not just KMS.

Reply:
Hi Chris,

I've amended the reference from KMS to Auto Unseal

`Rekey an Auto Unseal vault and encrypt the resulting recovery keys with PGP:`

Hopefully this makes more sense.

Kind regards,
Graham